website/templates/project/settings.mako

```diff
@@ -387,6 +411,8 @@
       window.contextVars.node.description = ${node['description'] | sjson, n };
       window.contextVars.node.nodeType = ${ node['node_type'] | sjson, n };
       window.contextVars.nodeCategories = ${ categories | sjson, n };
+      window.contextVars.wiki = window.contextVars.wiki || {};
```

Comment:
It was complaining when I did not create the wiki object first.  I created a wiki object first thinking there might be other wiki variables that are needed when I do the big overhaul, but I could make a simple boolean like window.contextVars.wikiEnabled if you want me to keep it simple.